lib/management.js

```diff
@@ -128,6 +128,7 @@ function patchConfiguration(instanceId, newConf, log, cb) {
                                 Boolean(l.details.serverSideEncryption),
                             awsEndpoint: l.details.endpoint ||
                                 's3.amazonaws.com',
+                            https: true,
```

Comment:
This should have been the default if `https` option is not defined.

Reply:
We use `setLocationConstraints()` to update the config in Orbit management.js https://github.com/scality/S3/blob/master/lib/management.js#L160 and so this function `locationConstraintAssert` never get called https://github.com/scality/S3/blob/master/lib/Config.js#L204. So I m guessing that for config management we have to specify it unless we change our checks here: https://github.com/scality/S3/blob/master/lib/data/locationConstraintParser.js#L63